apisix/plugins/file-logger.lua

```diff
@@ -138,10 +138,10 @@ local function write_file_data(conf, log_message)
             core.log.error("failed to write file: ", conf.path, ", error info: ", err)
         end
 
-        if conf.log_to_std then
-            local std_out, err = io_open(std_out_file, "w")
-            std_out:setvbuf("no")
+        if conf.log_to_stdout then
```

Comment:
This check should be in advance because here you open the `conf.path` and then close it finally. It's meaningless.

Reply:
I already modified this part of the code yesterday, please check the latest version.